cmd/system-probe/modules/network_tracer.go

```diff
@@ -292,6 +292,17 @@ func (nt *networkTracer) Register(httpMux *module.Router) error {
 		utils.WriteAsJSON(w, cache)
 	})
 
+	httpMux.HandleFunc("/debug/ebpf_btf_loader", func(w http.ResponseWriter, _ *http.Request) {
```

Comment:
This should be more system-probe wide, i.e. not confined to the network tracer module.

Reply:
Moved to ebpf module